parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -360,10 +360,6 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
 {{end}}
 {{if IsCustomCloudProfile }}
     AZURE_ENVIRONMENT_FILEPATH=/etc/kubernetes/azurestackcloud.json
-{{end}}
-{{if AnyAgentIsLinux}}
-    KUBELET_REGISTER_NODE=--register-node=true
```

Comment:
As far as I can tell this weird `--register-node=true` config inside this "any agent is Linux" block doesn't make sense. "true" is the default value for this, and we certainly never want to not register a node in the cluster.

Ref:

https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/